packages/pds/src/account-manager/helpers/account.ts

```diff
@@ -64,6 +64,41 @@ export const getAccount = async (
   return found || null
 }
 
+export const getAccounts = async (
+  db: AccountDb,
+  handlesAndDids: string[],
```

Comment:
since `getAccountInfos` only takes dids, you can probably simplify this by just taking an array of DIDs (no handles)